### PR TITLE
Add infrastructure setup for test host

### PR DESCRIPTION
**Pull request by outcatcher**
Implement playbook-driven test host setup

### CODE REVIEW

File: playbooks/files/get_key.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+def parse_params():
+    parser = ArgumentParser('OpenTelekomCloud S3 key downloader')
+    parser.add_argument('--key', '-k', default=None)
+    parser.add_argument('--output', '-o', default=None)
+    args = parser.parse_args()
+    return args
+
+
+def generate_private_key():
+    key = RSA.generate(2048)
```
Comment (anton-sidelnikov) [inline playbooks/files/get_key.py]: old version, need to use from master in customer-service-monitoring

Comment (outcatcher) [reply playbooks/files/get_key.py]: Done

**Comment by outcatcher** [comment]
Removing of terraform will be handled in #2

**Comment by outcatcher** [comment]
**Not ready for merge yet**

**Comment by outcatcher** [comment]
Need to split running each scenario to different playbook

**Review by lego963 — DISMISSED** [review]

File: playbooks/files/infrastructure/net.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
+
+resource "opentelekomcloud_vpc_v1" "vpc" {
+  cidr = "192.168.0.0/28"
+  name = "${local.prefix}_vpc"
+}
+
+resource "opentelekomcloud_vpc_subnet_v1" "subnet" {
+  cidr          = "192.168.0.0/29"
```
Comment (anton-sidelnikov) [inline playbooks/files/infrastructure/net.tf:8]: should be /30

File: playbooks/files/infrastructure/net.tf [hunk 3]
```diff
@@ -0,0 +1,25 @@
+
+resource "opentelekomcloud_vpc_v1" "vpc" {
+  cidr = "192.168.0.0/28"
```
Comment (anton-sidelnikov) [inline playbooks/files/infrastructure/net.tf:3]: should be /30

File: playbooks/roles/start_scenario/defaults/main.yml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+csm_home: '/home/linux/customer-service-monitoring'
```
Comment (anton-sidelnikov) [inline playbooks/roles/start_scenario/defaults/main.yml]: why this block duplicated in inventory?

Comment (outcatcher) [reply playbooks/roles/start_scenario/defaults/main.yml]: Replaced with required var list in `defaults` for each role

**Review by anton-sidelnikov — DISMISSED** [review]

**Review by lego963 — APPROVED** [review]

**Review by anton-sidelnikov — APPROVED** [review]

**Pull request merged**